### ISSUE TITLE
mC in other motifs and R10

### ISSUE THREAD

**Issue by yusmiatiliau** [hasindu2008/f5c#119]
Dear Hasindu,

May I know if there is any plan to include 5mC in other motifs for application in plant sample?
Also, is f5c compatible with data from R10 flowcells?

Thank you,
Cen

**Comment by hasindu2008** [comment]
Hi

We are currently generating models for supporting R10. But this is a work in progress and will take some time to come to the master branch. 

It would be interesting the to see how plant samples can be supported - but I am not much familiar with plant samples and do not really have any datasets. But I am positive to explore this if you can provide some guidance on example data sets etc.

**Comment by yusmiatiliau** [comment]
Hi,

Thank you for your response, and looking forward to the R10 support.
The main thing in plant is that unlike human where 5mC only present in CpG context, they are also present in CHG and CHH motifs.  Another tool, DeepSignal-Plant has model for this, but they currently do not support R10 data yet. ONT has a research remora model for this in bonito, but it would be breat to have comparison with other tools. Thanks again,

**Comment by hasindu2008** [comment]
@yusmiatiliau 
Finally a very crude model for r10.4.1 methylation is available. Do you happen to have some data for testing?

**Comment by yusmiatiliau** [comment]
Hi Hasindu,

Thank you for your update. Yes, I do have some data to test, that I have
called methylation from, using guppy with remora CpG model and bonito with
the all context model. I will check out your f5c model next week and do
some comparison.

Thanks again,
Cen

**Comment by hasindu2008** [comment]
Hi

I will be ok with either. Could you check if you can build f5c from the r10 branch?

```
git clone https://github.com/hasindu2008/f5c/ -b r10
autoreconf             
scripts/install-hts.sh  
./configure
make 
```

Then, run the test script for r10:
```
scripts/test_lsk114.sh
head test/hg2_lsk114_reads_1000/meth.tsv # to see if there are any calls
```

Then, you should specify the options models manually using options `--kmer-model test/r10-models/r10.4.1_400bps.nucleotide.9mer.template.model --meth-model test/r10-models/r10.4.1_400bps.cpg.9mer.template.model` to f5c.

**Comment by yusmiatiliau** [comment]
Hi Hasindu,

I will have a try and let you know how it goes. So this model would be
calling 5mC on CpG context only, am I right? Is there a possibility to
expand to other contexts too?

Thank you,
Cen

**Comment by hasindu2008** [comment]
This is for 5mc in cpg context. It is definitely possible to expand, but it is a bit of work. Given remora is already supporting those, I wonder if it is worth putting effort.

**Comment by hasindu2008** [comment]
@yusmiatiliau I have released some binaries f5c-v1.2-beta with the r10 models inbuilt. I tested on some human samples and get close to 0.9 correlation with bisulphite (without filtering). Do you happen to have any plant data to test?

**Comment by hasindu2008** [comment]
Will close this issue for now, feel free to reopen if you have any issues.
